File: modl 4/python/PRAK405-2310817210027-MuhammadErzaRaihan.py
def main():
    x, n = map(int, input(" ").split())
    total = 0

    for i in range(1, x + 1):
        baris = 0

        for j in range(i, 0, -1):
            baris += j * n
            if j == i:
                print(f"({j} * {n})", end="")
            else:
                print(f" + ({j} * {n})", end="")

        print(f" = {baris}")
        total += baris

    print(total)

if __name__ == "__main__":
    main()

File: modl 4/C/PRAK405-2310817210027-MuhammadErzaRaihan.c
#include <stdio.h> 
  
 int main () { 
     int x, n; 
     int total = 0; 
   
     scanf("%d %d", &x, &n); 
  
     for (int i = 1; i <= x; i++) { 
  
         int baris = 0; 
  
         for (int j = i; j > 0; j--) { 
  
             baris += j * n; 
             if (j == i) { 
                 printf("(%d * %d)", j, n); 
             }  
             else { 
                 printf(" + (%d * %d)", j, n); 
             } 
  
         } 
  
         printf(" = %d\n", baris); 
         total += baris; 
     } 
  
     printf("%d\n", total); 
     return 0;
 }